autogtp/Production.cpp

```diff
@@ -117,26 +118,25 @@ Production::Production(const int gpus,
 }
 
 bool Production::updateNetwork() {
-    auto retries = 0;
```

Comment:
I'm trying to learn what c++ styles you like and I didn't know here,
I prefer for loops when the loop count is constant (baring early return / break).
I'll revert this if you don't agree (and stick with the do while that you seem to prefer in many places)

Reply:
I don't really care.

(Note that I didn't wrote most of this code :-)